src/Common/TurbulenceStatistics/kinetic_energy.jl

```diff
@@ -0,0 +1,62 @@
+function average_kinetic_energy_and_dissipation_writer(
+    solver_config,
+    driver_config,
+    E_0,
+    iter,
+    outfile,
+    nor,
+)
+    mpicomm = MPI.COMM_WORLD
+    mpirank = MPI.Comm_rank(mpicomm)
+    if mpirank == 0
+        E_k0 = average_kinetic_energy(solver_config, driver_config, nor)
```

Comment:
This won't work on more than one rank since you're calling `MPI_Reduce()` in `average_kinetic_energy()` below, but only from rank 0.

Reply:
Good point, I'll fix that